core/bones/sortindex.py

```diff
@@ -1,5 +1,5 @@
 from viur.core.bones.numeric import NumericBone
-import time, typing
+import time, typing,sys
```

Comment:
This is no longer necessary.

```suggestion
import time, typing
```